dydx/dydxPresenters/dydxPresenters/_v4/MarketInfo/dydxMarketInfoViewBuilder.swift

```diff
@@ -174,7 +174,7 @@ private class dydxMarketInfoViewPresenter: HostedViewPresenter<dydxMarketInfoVie
             guard let marketId = self?.marketId,
                   let assetId = marketMap[marketId]?.assetId,
                   let asset = assetMap[assetId] else { return }
-            if asset.tags?.contains("Prediction Market") != true && !Self.hidePredictionMarketsNotice {
+            if asset.tags?.contains("Prediction Market") == true && !Self.hidePredictionMarketsNotice {
```

Comment:
this is a change to stop forcing the prediction markets notice screen. Was forcing this for the test build.